testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -137,6 +147,25 @@ def test_volume(self):
                             err_msg="wrong volume for unitcell (no unitcell "
                             "in DCD so this should be 0)")
 
+    def test_timeseries_slicing(self):
+        # check that slicing behaves correctly
+        # should fail before issue #914 resolved
+        x = []
+        for i in range(5):
```

Comment:
It is probably not needed to test all values of start, stop, and slice with such triple loop. Most of the test values will provide no additional insight on possible bugs, and you are not testing `None` as a value.

Instead of the triple loop, I would have an explicit list of `(start, stop, step)` tuples, like in @richardjgowers example. This allows to target relevant values, and to comment what that are testing (all default, backward slicing, stop beyond the end, step beyond the end...).